pywt/tests/test_multilevel.py

```diff
@@ -50,6 +52,56 @@ def test_swt_decomposition():
     assert_(pywt.swt_max_level(len(x)) == 3)
 
 
+def test_swt_iswt_integration():
+    """
+    This function performs a round-trip swt/iswt transform test on
+    all available types of wavelets in PyWavelets - except the 'dmey'
+    wavelet. The latter has been excluded because it does not produce
+    very precise results. This is likely due to the fact that the
+    'dmey' wavelet is a discrete approximation of a continuous
+    wavelet. All wavelets are tested up to 3 levels. The test
+    validates neither swt or iswt as such, but it does ensure
+    that they are each other's inverse.
+    """
+    max_level = 3
+    wavelets = pywt.wavelist()
+    if 'dmey' in wavelets:
+        wavelets.remove('dmey')  # The 'dmey' wavelet seems to be a bit special - disregard it for now
```

Comment:
Yes please. PEP8 says "Use inline comments sparingly". In almost all cases you want the comment above the code you're commenting on. (plus here the line is >80 chars)